src/os/ui/query/queryhandler.js

```diff
@@ -457,3 +505,20 @@ os.ui.query.QueryHandler.prototype.wrap = function(filter, order, excludes, grou
 os.ui.query.QueryHandler.prototype.wrapAll = function(filter) {
   return this.filterFormatter ? this.filterFormatter.wrapAll(filter) : '';
 };
+
+
+/**
+ * @param {os.ui.query.QueryHandler?} opt_qh another query handler to clone to
```

Comment:
This is better named as `opt_result` or `opt_clone`. Also any optional param should be `{Type=}` not `{Type?}`.

Reply:
Good catch!